src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyndto/BeregningBarnetilsynRequest.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.ef.sak.beregning
```

Comment:
Feil package? 
Burde man splitte opp koden i
beregning
 - overgangsstønad
 - barnetilsyn
 
I steder for å ha et package som heter `barnetilsyndto`

Reply:
Vi lurte på om vi skulle ha to pakker. Tar en prat om det.
Fikser uansett feil package 👍

Reply:
Jeg flytter alt som har med barnetilsyn til egen barnetilsynpakke nå (denne pr'en)
Lager ny pr på å flytte resten av det som ligger igjen i beregning til overgangsstønad-pakke